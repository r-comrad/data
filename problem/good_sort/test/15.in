42
703624 5318657 8611182 1835442 6343041 8446513 9786459 5586215 8458473 5623766 9490874 1179397 1257883 6456269 3302648 6302446 9352602 8805146 7654375 9940082 3307825 1380532 9092080 7947194 2560964 9450422 6433037 8882565 2227468 1607433 1935871 6947761 1353513 6790025 3023782 9202364 9144095 4623532 1034631 1114372 1227890 9093108 