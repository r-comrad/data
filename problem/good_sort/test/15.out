9450422 6790025 1380532 2560964 9490874 9940082 8611182 9352602 3302648 6302446 8882565 9202364 9092080 9093108 1353513 703624 4623532 6433037 6343041 5623766 3023782 1034631 1114372 7654375 9144095 8805146 1835442 1935871 8446513 5586215 9786459 6456269 3307825 1607433 6947761 2227468 1257883 1227890 7947194 5318657 8458473 1179397 